fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -147,9 +139,20 @@ export const LocationSuggest = forwardRef<
           schemeId,
           geoLocation: input,
         },
+        context,
       });
     };
 
+    useEffect(() => {
```

Comment:
Kinda gross but it the onCompleted hook got removed.


This issue was linked: https://github.com/apollographql/apollo-client/issues/12352

Reply:
Just in Sam condones the use of `useEffect` 😆 🧌